### PR TITLE
Full project implementation for JS best practices assesment 

### PR DESCRIPTION
**Pull request by bruk19**
- Completed the full project requirements

### CODE REVIEW

**Comment by jadibdev** [comment]
- Kindly remove comments from your code as it makes your code cleaner
https://github.com/bruk19/to-do-list-review-/blob/275fe0ab9a7fd31cc02043b24377425167484864/src/remove.js#L11-L18

- Kindly consider using section instead of div as it is more semantic
https://github.com/bruk19/to-do-list-review-/blob/275fe0ab9a7fd31cc02043b24377425167484864/src/index.html#L9-L17

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<body>
+    <main><div class="listContainer">
+        <div class="title"><h3>Today's To Do</h3><img class = 'imgRefresh'  alt = 'refresh'></div>
+        <form class = "addTask">
+          <input type="text" class="p1" placeholder="Add to your list...">
+          <input type="submit" value = '' class = "submit"><img class = 'imgSubmit' alt = 'return'>
+        </form>
+        <ul class="to-do-list"></ul>
+        <div class="clear">Clear all completed</div>
+      </div></main>
```
Comment (jadibdev) [inline src/index.html:17]: - Kindly consider using section instead of div as it is more semantic

File: src/remove.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  for (let i = index - 1; i < list.length; i += 1) {
+    /*
+    let ind = i + 2;
+   let nextCheck = document.querySelector(`#task-${ind}`)
+    nextCheck.attributes.id = `task-${i + 1}`;
+    nextCheck.value -= 1;
+    list[i].index -= 1;
+    localStorage.setItem('tasks', JSON.stringify(list));
+    */
```
Comment (jadibdev) [inline src/remove.js:18]: - Kindly remove comments from your code as it makes your code cleaner

**Review by jadibdev — COMMENTED** [review]
Great use of modular Javascript, the code is mostly DRY, keep up the good work.

File: src/remove.js (hunk 2 above)
Comment (bruk19) [reply src/remove.js:18]: Thank you for your feedback. I'll have this in mind.

**Pull request merged**